libs/util/src/lib/calculate-timeout-timer.ts

```diff
@@ -0,0 +1,7 @@
+const averageWPM = 50;
+
+export const calculateTimeoutTimer = async (text: string) => {
```

Comment:
Can you please add a jsdoc explanation for this? 
What's the unit of `timeoutDuration`?

Reply:
Completed docs  in PR #14 under commit 4d6c57dec3d53ee4b0ea4ee596fa7368c3a0833d